src/main/java/rip/bolt/ingame/commands/TournamentAdminCommands.java

```diff
@@ -35,6 +45,47 @@ public void register(CommandSender sender, TournamentTeamManager teamManager, @T
         sender.sendMessage(ChatColor.YELLOW + "Added team " + team.getName() + "!");
     }
 
+    @Command(aliases = "add", desc = "Add a player to a team", usage = "<team> <player>", perms = "ingame.staff")
+    public void add(CommandSender sender, TournamentTeamManager teamManager, String teamName, @Text String userName) {
+        TournamentTeam team = ConfigTeamParser.getInstance().getTeam(teamName);
+
+        if (team == null) { // TODO move to provider
+            sender.sendMessage(ChatColor.RED + "Team not found!");
+            return;
+        }
+
+        UUID uuid = UUIDFetcher.getUUID(userName);
```

Comment:
This is a blocking operation and will freeze the server until the server responds

Reply:
@Pablete1234 I updated it, I wasnt sure if its better to move the async stuff to the uuid fetcher class and pass the sender when calling it, or just wrap it in the command class itself.